src/orders/ordersClient.ts

```diff
@@ -53,7 +53,7 @@ export type CreateOrderInputType = {
   [CreateOrderInputFields.COLUMN_SCENARIO]?: scenarioType;
   [CreateOrderInputFields.COLUMN_SCHEDULE_DATE]?: string;
   [CreateOrderInputFields.COLUMN_PRODUCTS]: Array<{
-    [CreateOrderInputFields.COLUMN_ARROW_SPHERE_PRICE_BAND_SKU]: string;
+    [CreateOrderInputFields.COLUMN_ARROW_SPHERE_PRICE_BAND_SKU]?: string;
     [CreateOrderInputFields.COLUMN_SKU]: string;
```

Comment:
sku field is not mandatory, so you need either arrowSpherePriceBandSku or sku

```suggestion
    [CreateOrderInputFields.COLUMN_SKU]?: string;
```